### DISCUSSION TITLE
can we call the kedro Pipeline through some REST client calls

### DISCUSSION THREAD

**Discussion by prk2331** [kedro-org/kedro#3368] [Idea]
Hi team,
      my aim is to host the kedro to another machine for workflows, and then call the work flows (pieplines) in my other machine with Python client SDK or through REST calls, if this is possible can someone please provide me the code snippet 
      
      Thanks

**Comment by astrojuanlu** [comment]
Hi @prk2331 , please subscribe to this thread https://github.com/kedro-org/kedro/issues/143 we are discussing what should be the best way to approach this
